index.html

```diff
@@ -1,76 +1,200 @@
+
+    <div class="row">
+      <div class="col-md-4">
+        <h3>
+          <a href="/services/mobile-ci-cd/"><i class="fa fa-cogs"></i> CI CD</a>
+        </h3>
+        <p class="alt">
+          Maecenas ut magna aliquet, facilisis odio ut, sollicitudin nunc. 
+        </p>
+        <p>
+          Lorem ipsum dolor sit amet, consectetur adipiscing elit. Maecenas magna ante, cursus sit amet accumsan at, sodales blandit.
+        </p>
+        <p>
+          <a href="/services/mobile-ci-cd/" class="btn btn-primary">Learn about <strong>CI CD</strong></a>
+        </p>
+      </div>
+
+      <div class="col-md-4">
+        <h3>
+          <a href="/services/metrics/"><i class="fa fa-bar-chart"></i> Metrics</a>
+        </h3>
+        <p class="alt">
+          Maecenas ut magna aliquet, facilisis odio ut, sollicitudin nunc. 
+        </p>
+        <p>
+          Lorem ipsum dolor sit amet, consectetur adipiscing elit. Maecenas magna ante, cursus sit amet accumsan at, sodales blandit.
+        </p>
+        <p>
+          <a href="/services/metrics/" class="btn btn-primary">Learn about <strong>Metrics</strong></a>
+        </p>
+      </div>
+
+      <div class="col-md-4">
+        <h3>
+          <a href="/services/push/"><i class="fa fa-external-link"></i> Push</a>
+        </h3>
+        <p class="alt">
+          Maecenas ut magna aliquet, facilisis odio ut, sollicitudin nunc. 
+        </p>
+        <p>
+          Lorem ipsum dolor sit amet, consectetur adipiscing elit. Maecenas magna ante, cursus sit amet accumsan at, sodales blandit.
+        </p>
+        <p>
+          <a href="/services/push/" class="btn btn-primary">Learn about <strong>Push</strong></a>
+        </p>
+      </div>
+
+      <div class="col-md-4">
+        <h3>
+          <a href="/services/keycloak/"><i class="fa fa-lock"></i> Identity Management</a>
+        </h3>
+        <p class="alt">
+          Maecenas ut magna aliquet, facilisis odio ut, sollicitudin nunc. 
+        </p>
+        <p>
+          Lorem ipsum dolor sit amet, consectetur adipiscing elit. Maecenas magna ante, cursus sit amet accumsan at, sodales blandit.
+        </p>
+        <p>
+          <a href="/services/keycloak/" class="btn btn-primary">Learn about <strong>Identity Management</strong></a>
+        </p>
+      </div>
+
+      <div class="col-md-4">
+        <h3>
+          <a href="/services/crc/"><i class="fa fa-plug"></i> CRC</a>
+        </h3>
+        <p class="alt">
+          Maecenas ut magna aliquet, facilisis odio ut, sollicitudin nunc. 
+        </p>
+        <p>
+          Lorem ipsum dolor sit amet, consectetur adipiscing elit. Maecenas magna ante, cursus sit amet accumsan at, sodales blandit.
+        </p>
+        <p>
+          <a href="/services/crc/" class="btn btn-primary">Learn about <strong>CRC</strong></a>
+        </p>
+      </div>
+
+      <div class="col-md-4">
+        <h3>
+            <a href="#">Learn More</a>
```

Comment:
I'm not sure about having `Learn More` here. For me, it doesn't make sense to be added as an item under `Aerogear Services` especially when we already have a `learn more` button to each of the available services.

Reply:
Yeah I see what you mean. I added it because it looked weird to have a gaping hole. The intention would be to link it to the Mobile Services overview page later